### ISSUE TITLE
RootCAs are not set in the call to NewServiceClient

### ISSUE THREAD

**Issue by YuxinZhou** [vmware/govmomi#1553]
Hi there, 

I am creating a vSAN service client from the vim client, but the RootCAs are not set in the call to NewServiceClient. So I need to manually set RootCAs again. Is this the expected behavior?
```go
vsanClient := vimClient.NewServiceClient(vsanPath, vsanNamespace)
// need to do this even RootCA is set for vim client
vsanClient.SetRootCAs(e.Parent.TLSCA)
```
Thanks a lot!

**Comment by dougm** [comment]
Hi @YuxinZhou , what revision are you using?  This should be fixed by #1498

**Comment by YuxinZhou** [comment]
Thanks for replying. Switched to v0.21.0 and resolved!
